core/Util/plPNG.h

```diff
@@ -17,7 +17,8 @@
 #ifndef _PLPNG_H
 #define _PLPNG_H
 
-#include "PRP/Surface/plMipmap.h"
+#include "PlasmaDefs.h"
+#include "Debug/hsExceptions.hpp"
```

Comment:
```suggestion
```

No exceptions thrown here... This should probably be included in the .cpp file.

Reply:
hsPNGException is unhappy with that.